.github/workflows/pr-check.yaml

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: pr-check
+
+on: [pull_request]
+
+jobs:
+  lint-format-unit:
+    name: linter, formatters and unit tests / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 40
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2022, ubuntu-22.04]
```

Comment:
line 42 we see mac/Linux but here there is only Ubuntu, no macOS so I don't know if you omitted macOS on purpose or not

Reply:
i copied the portion i needed from the podman-desktop repo. I don't remember if there was some reasoning behind it. But i guess i can add the mac machine as well for what we do atm

Reply:
Done 👍